server/schedulers/evict_leader.go

```diff
@@ -189,6 +200,11 @@ func newEvictLeaderScheduler(opController *schedule.OperatorController, conf *ev
 	}
 }
 
+// EvictStores returns the IDs of the evict-stores.
+func (s *evictLeaderScheduler) EvictStoreIDs() []uint64 {
```

Comment:
need lock?

Reply:
We do not need a lock here since the method of `s.conf.getStores()` is thread safe